setup.cfg

```diff
@@ -43,11 +43,10 @@ include_package_data = True
 
 [options.extras_require]
 full =
-  nibabel >= 2.2
+  nibabel >= 2.5
   pybv >= 0.6
   matplotlib >= 3.1.0
   pandas >= 0.24.0
-  openneuro-py >= 2021.8
```

Comment:
I remove openneuro-py here because we only need it for the documentation (like nilearn, etc.)

Reply:
well full means also doc to me. It's also pure Python package. Otherwise we can do a [doc] install target

Reply:
okay, but then we'd need a couple of more packages there as well right? sphinx, nilearn, sphinx-copybutton, ... https://github.com/mne-tools/mne-bids/blob/ff5cdeda3772bb8e7a82fcd1caf4a5584edc40ec/doc/requirements.txt#L1-L10

Reply:
To me, full means full functionality for a user, but no dev deps. Building docs is a dev operation. I don't think full needs to support this

Reply:
I rather agree with @hoechenberger here - but Richard: why would you then include openneuro-py in "full"?